docker-compose-mon.yml

```diff
@@ -0,0 +1,151 @@
+      LOTUS_TEST:
+      LOTUS_API:
+      LOTUS_TOKEN:
+    depends_on:
+      singularity_admin_init:
+        condition: service_completed_successfully
+
+  singularity_deal_pusher:
+    image: ghcr.io/data-preservation-programs/singularity${SINGULARITY_REF}
+    volumes:
+      - motion-singularity-volume:/usr/src/app/storage
+    command: run deal-pusher
+    environment:
+      DATABASE_CONNECTION_STRING: postgres://${SINGULARITY_DB_USER:-postgres}:${SINGULARITY_DB_PASSWORD:-postgres}@db:5432/${SINGULARITY_DB_NAME:-singularity}
+      LOTUS_TEST:
+      LOTUS_API:
+      LOTUS_TOKEN:
+    depends_on:
+      singularity_admin_init:
+        condition: service_completed_successfully
+
+  singularity_deal_tracker:
+    image: ghcr.io/data-preservation-programs/singularity${SINGULARITY_REF}
+    volumes:
+      - motion-singularity-volume:/usr/src/app/storage
+    command: run deal-tracker
+    environment:
+      DATABASE_CONNECTION_STRING: postgres://${SINGULARITY_DB_USER:-postgres}:${SINGULARITY_DB_PASSWORD:-postgres}@db:5432/${SINGULARITY_DB_NAME:-singularity}
+      LOTUS_TEST:
+      LOTUS_API:
+      LOTUS_TOKEN:
+    depends_on:
+      singularity_admin_init:
+        condition: service_completed_successfully
+
+  singularity_content_provider:
+    image: ghcr.io/data-preservation-programs/singularity${SINGULARITY_REF}
+    command: run content-provider --http-bind :7778
+    volumes:
+      - motion-singularity-volume:/usr/src/app/storage
+    ports:
+      - 7778:7778
+    environment:
+      DATABASE_CONNECTION_STRING: postgres://${SINGULARITY_DB_USER:-postgres}:${SINGULARITY_DB_PASSWORD:-postgres}@db:5432/${SINGULARITY_DB_NAME:-singularity}
+      LOTUS_TEST:
+      LOTUS_API:
+      LOTUS_TOKEN:
+    depends_on:
+      singularity_admin_init:
+        condition: service_completed_successfully
+
+  motion:
+    image: ghcr.io/filecoin-project/motion:main
+    entrypoint: motion --experimentalSingularityStore --experimentalRemoteSingularityAPIUrl=singularity_api:9090 --experimentalSingularityContentURLTemplate=${SINGULARITY_CONTENT_PROVIDER_DOMAIN:-http://singularity_content_provider:7778}/piece/{PIECE_CID}
+    ports:
+      - 40080:40080
+    environment:
+      - MOTION_STORE_DIR=/usr/src/app/storage
+      - LOTUS_TEST
+      - LOTUS_API
+      - LOTUS_TOKEN
+      - MOTION_STORAGE_PROVIDERS
+      - MOTION_PRICE_PER_GIB_EPOCH
+      - MOTION_PRICE_PER_GIB
+      - MOTION_PRICE_PER_DEAL
+      - MOTION_DEAL_START_DELAY
+      - MOTION_DEAL_DURATION
+      - MOTION_SINGULARITY_MAX_CAR_SIZE
+      - MOTION_SINGULARITY_PACK_THRESHOLD
+      - MOTION_SINGULARITY_SCHEDULE_CRON
+      - MOTION_SINGULARITY_SCHEDULE_DEAL_NUMBER
+      - MOTION_WALLET_KEY
+      - MOTION_VERIFIED_DEAL
+    volumes:
+      - motion-singularity-volume:/usr/src/app/storage
+    depends_on:
+      singularity_api:
+        condition: service_healthy
+
+  dozzle:
+    container_name: dozzle
```

Comment:
Could you explain the reason of choosing dozzle over other "built-in" logging driver such as
https://docs.docker.com/config/containers/logging/fluentd/

Reply:
Also, I wonder if we should include our preference of logging solution in this repo, or should we leave it blank so user can choose to use their own preferred logging solution.